FFMediaToolkit/Common/MediaType.cs

```diff
@@ -1,9 +1,9 @@
-﻿namespace FFMediaToolkit.Common.Internal
+﻿namespace FFMediaToolkit.Common
 {
     /// <summary>
     /// Represents the multimedia stream types.
     /// </summary>
-    internal enum MediaType
+    public enum MediaType
```

Comment:
To me it made sense to expose this feature because it wasn't really even used internally.  Its more of a thing that the user would want to choose anyway, whether to discard audio/video data or not.

Reply:
I guess it's find, but is it actually exposed in any way other than making the type public?

Reply:
Yes, it is a property of the MediaOptions class. 😀

Reply:
Ah wait, I believe I have confused `MediaMode` and `MediaType`.  However, both should still be public, as `MediaType` is exposed via `StreamInfo`.